src/taggle.js

```diff
@@ -506,12 +511,15 @@
             span = li.querySelector('.taggle_text');
             text = span.innerText || span.textContent;
 
+            if (!settings.onBeforeTagRemove(e, text)){
```

Comment:
`settings.onBeforeTagRemove(e, text)` will work just fine without this `if` block.